src/components/textEditor/index.tsx

```diff
@@ -159,21 +55,29 @@ const TextEditor: React.FC = (): JSX.Element => {
         <Button submitFunction={handleSubmit} text="Hint" />
         <Button submitFunction={handleSubmit} text="Run" />
       </TerminalHeader>
-      <Editor>
-        <LineIndicator lineCount={lineCount} />
-        <Terminal
-          type="text"
-          onBlur={removeSuggestion}
-          onChange={handleChange}
-          spellCheck="false"
-          placeholder="Write your code here."
-          value={userInput}
-          lineCount={lineCount}
-        />
-      </Editor>
-      <SuggestBox suggestion={suggestion}>
-        <h1>{suggestion}</h1>
-      </SuggestBox>
+      <AceEditor
+        style={{
+          position: "relative",
+          marginTop: "1%",
+          height: "90%",
+          width: "99.9%",
+          backgroundColor: background
+        }}
+        fontSize="18px"
```

Comment:
Please change the fontsize to 16px and the font-family to "Source Code Pro" to reflect the redesign.